src/Reservations/Reservations/help/Az.Reservations.md

```diff
@@ -27,7 +27,7 @@ Get `ReservationOrder`
 Get list of applicable `ReservationOrder` Ids.
 
 ### [Get-AzReservationQuote](Get-AzReservationQuote.md)
-{{ Fill in the Synopsis }}
```

Comment:
Please also update the synopsis in https://github.com/Azure/azure-powershell/blob/c798ce5d78cf19154c22fd338b377b7e65a01917/src/Reservations/Reservations/help/Get-AzReservationQuote.md